frame/evm/src/mock.rs

```diff
@@ -131,10 +131,14 @@ impl FindAuthor<H160> for FindAuthorTruncated {
 }
 const BLOCK_GAS_LIMIT: u64 = 150_000_000;
 const MAX_POV_SIZE: u64 = 5 * 1024 * 1024;
+/// The maximum storage growth per block in bytes (40 Kb).
+const MAX_STORAGE_GROWTH: u64 = 40 * 1024;
+const GAS_LIMIT_STORAGE_GROWTH_RATIO: u64 = 15_000_000u64.saturating_div(MAX_STORAGE_GROWTH);
```

Comment:
And this, use `BLOCK_GAS_LIMIT` directly.

Reply:
Done.